hermes-console/static/js/console/owner/OwnerName.js

```diff
@@ -0,0 +1,22 @@
+angular.module('hermes.owner')
+    .controller('nameCtrl', ['$scope', 'OwnerRepository', function ($scope, ownerRepository) {
```

Comment:
In other places we use xController, not xCtrl. I think we should stick with the former.